recipes/SimpleASTParserLog/README.md

```diff
@@ -0,0 +1,15 @@
+# SimpleASTParserLog
+
+This recipe shows how to run a simple "Hello world!" program in either the node.js or web browser console.
```

Comment:
Looks like remaining todos are to just update this file and run the readme script.

Reply:
Thanks for the reminder. I pushed those changes just now.